sdk/storage/Azure.Storage.Files.DataLake/samples/Sample01b_HelloWorldAsync.cs

```diff
@@ -0,0 +1,424 @@
+                DataLakeDirectoryClient deepest = await fourth.CreateSubDirectoryAsync("e");
+
+                // Upload a DataLake file named "file"
+                DataLakeFileClient file = deepest.GetFileClient("file");
+                await file.CreateAsync();
+                using (FileStream stream = File.OpenRead(originalPath))
+                {
+                    await file.AppendAsync(stream, 0);
+                }
+
+                // Keep track of all the names we encounter
+                List<string> names = new List<string>();
+                AsyncPageable<PathItem> response = filesystem.ListPathsAsync(recursive: true);
+                IList<PathItem> paths = await response.ToListAsync();
+                foreach (PathItem pathItem in paths)
+                {
+                    names.Add(pathItem.Name);
+                }
+
+                // Verify we've seen everything
+                Assert.AreEqual(10, paths.Count);
+                Assert.Contains("first", names);
+                Assert.Contains("second", names);
+                Assert.Contains("third", names);
+                Assert.Contains("fourth", names);
+                Assert.Contains("first/a", names);
+                Assert.Contains("first/b", names);
+                Assert.Contains("second/c", names);
+                Assert.Contains("second/d", names);
+                Assert.Contains("fourth/e", names);
+                Assert.Contains("fourth/e/file", names);
+            }
+            finally
+            {
+                // Clean up after the test when we're finished
+                await filesystem.DeleteAsync();
+            }
+        }
+
+        /// <summary>
+        /// Trigger a recoverable error.
+        /// </summary>
+        [Test]
+        public async Task ErrorsAsync()
+        {
+            // Make StorageSharedKeyCredential to pass to the serviceClient
+            string storageAccountName = StorageAccountName;
+            string storageAccountKey = StorageAccountKey;
+            Uri serviceUri = StorageAccountBlobUri;
+            StorageSharedKeyCredential sharedKeyCredential = new StorageSharedKeyCredential(storageAccountName, storageAccountKey);
+
+            // Create DataLakeServiceClient using StorageSharedKeyCredentials
+            DataLakeServiceClient serviceClient = new DataLakeServiceClient(serviceUri, sharedKeyCredential);
+
+            // Get a reference to a container named "sample-filesystem-errorsasync" and then create it
+            DataLakeFileSystemClient filesystem = serviceClient.GetFileSystemClient("sample-filesystem-errorsasync");
+            await filesystem.CreateAsync();
+            try
+            {
+                // Try to create the filesystem again
+                await filesystem.CreateAsync();
+            }
+            catch (RequestFailedException ex)
+                when (ex.ErrorCode == Constants.Blob.Container.AlreadyExists)
```

Comment:
This is an internal constant customers can't access.  Is there a DataLakeErrors.FileSystemAlreadyExists or similar?